install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -216,13 +218,47 @@ spec:
     responseCode: response.code | 0
     responseSize: response.size | 0
     requestSize: request.size | 0
+    requestId: request.headers["x-request-id"] | ""
+    clientTraceId: request.headers["x-client-trace-id"] | ""
     latency: response.duration | "0ms"
     connectionMtls: connection.mtls | false
     userAgent: request.useragent | ""
-    responseTimestamp: response.time
+    responseTimestamp: response.time | timestamp("2017-01-01T00:00:00Z")
     receivedBytes: request.total_size | connection.received.bytes | 0
     sentBytes: response.total_size | connection.sent.bytes | 0
     referer: request.referer | ""
+    xForwardedFor: request.headers["x-forwarded-for"] | "0.0.0.0"
+  monitored_resource_type: '"global"'
+---
+apiVersion: "config.istio.io/v1alpha2"
+kind: logentry
+metadata:
+  name: tcpaccesslog
+  namespace: {{ .Release.Namespace }}
+spec:
+  severity: '"Info"'
+  timestamp: context.time | context.timestamp | timestamp("2017-01-01T00:00:00Z")
```

Comment:
So what is the difference between context.time and context.timestamp?

Reply:
Don't see context.timestamp anymore.. thanks for asking.. removed it.